### PR TITLE
Update inactive display UI

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update inactive display UI

Add configuration options and email form inline
Improve email form

[stage-2]

## Motivation and Context
Update UI as per design

## How Has This Been Tested?
Tested changes locally. Updated tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -48,15 +48,23 @@
 </div> 
 
 <div class="border-container text-center u_margin-md-top" ng-show="!displayService.statusLoading && (display | status) === 'notinstalled'">
```
Comment (ezequielc) [inline web/partials/displays/display-fields.html:50]: @alex-deaconu I noticed that the Not Activated panel shows while the page is loading. Could you hide it if display is loading (in a future PR) Thanks!

File: web/scss/sections/displays.scss [hunk 2]
```diff
@@ -92,6 +92,15 @@
 
 }
 
+.display-instructions-panel {
```
Comment (ezequielc) [inline web/scss/sections/displays.scss:95]: @alex-deaconu Don't forget to handle mobile resolution as well. Thanks

File: web/partials/displays/display-fields.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/displays/display-fields.html:50]: @ezequielc good suggestion will fix. Thanks!

File: web/scss/sections/displays.scss (hunk 2 above)
Comment (alex-deaconu) [reply web/scss/sections/displays.scss:95]: @ezequielc good catch, will fix. Thanks!

**Pull request merged**